src/profiles.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface ResourceCapability {
+    type: string;
+    supportedProfile: string[];
+}
+
+export interface ResourceCapabilityStatement {
+    [resourceType: string]: ResourceCapability;
+}
```

Comment:
Could you also move this interfaces to the routing package? It looks like they are only used internally by the routing package and not anywhere else.

Going forward we'd like to keep the interface package to only the minimum required for collaboration between packages.

Reply:
Got it, I moved these interfaces to the routing package now.